src/sciline/scheduler.py

```diff
@@ -77,7 +77,17 @@ def __init__(self, scheduler: Optional[Callable[..., Any]] = None) -> None:
             self._dask_get = scheduler
 
     def get(self, graph: Graph, keys: List[Key]) -> Any:
-        dsk = {tp: (provider, *args) for tp, (provider, args) in graph.items()}
+        from dask.utils import apply
+
+        dsk = {
+            tp: (
+                apply,
+                provider,
+                list(args.args),
+                (dict, [[key, val] for key, val in args.kwargs]),
```

Comment:
A comment would be useful, it is not obvious for a reader what is going on here.